bluebrain/repo-bluebrain/packages/py-morphio/package.py

```diff
@@ -26,4 +26,4 @@ class PyMorphio(PythonPackage):
     depends_on('ninja', type='build')
     depends_on('cmake@3.2:', type='build')
     depends_on('py-numpy@1.14.1:', type='run')
-    depends_on('hdf5', type=('build', 'run'))
+    depends_on('hdf5')
```

Comment:
Is this diff intentional?

Reply:
Yes, pretty much. HDF5 is not a `run` dependency, but a link one.